docs/layers/text-layer.md

```diff
@@ -8,6 +8,53 @@
 
 The text layer renders text labels on the map using texture mapping. This Layer is extended based on [Icon Layer](/docs/layers/icon-layer.md) and wrapped using [Composite Layer](/docs/api-reference/composite-layer.md).
 
+##### Auto packing fontAtlas
+
+Auto pack required `characterSet` into a shared texture `fontAtlas`.
+
+```js
+import DeckGL from 'deck.gl';
+import TextLayer from './text-layer';
+
+const App = ({data, viewport}) => {
+  /**
+   * Data format:
+   * [
+   *   {name: 'Colma (COLM)', address: '365 D Street, Colma CA 94014', coordinates: [-122.466233, 37.684638]},
+   *   ...
+   * ]
+   */
+
+  const layer = new TextLayer({
+    id: 'text-layer',
+    data,
+
+    // used in generating `fontAtlas`
+    fontSize: 64,
+
+    pickable: true,
+    getPosition: d => d.coordinates,
+    getText: d => d.name,
+    getSize: 32,
+    getAngle: 0,
+    getTextAnchor: 'middle',
+    getAlignmentBaseline: 'center',
+    onHover: ({object, x, y}) => {
+      const tooltip = `${object.name}\n${object.address}`;
+      /* Update tooltip
+         http://deck.gl/#/documentation/developer-guide/adding-interactivity?section=example-display-a-tooltip-for-hovered-object
+      */
+    }
+  });
+
+  return <DeckGL {...viewport} layers={[layer]} />;
+};
+```
+
+### Rendering With sdf (Signed Distance Fields)
+
+This is slower than `Auto packing fontAtlas` but can provide a better antialiased look.
```

Comment:
- Move the link to SDF paper here
- It's not really about antialiasing. try `this font is slower to generate but can provide a sharper look when used with very large or small font sizes.`